src/components/main-menu/_main-menu.scss

```diff
@@ -0,0 +1,93 @@
+.header-nav {
+  display: none;
+
+  li {
+    list-style: none;
+  }
+
+  @media screen and (min-width: $large-screen) {
+    display: block;
+
+    .menu--list {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  .menu--list__item {
+    margin-left: px-to-rem(48px);
+    position: relative;
+
+    > ul {
+      display: none;
+    }
+
+    // Style dropdown buttons
+    button {
+      display: block;
+
+      &::after {
+        background-image: url('../files/icons/menu-dropdown.svg');
```

Comment:
Can you switch the arrows, currently showing arrow-right then click and arrow-down. Should it not be arrow-down click then arrow-up.